api-catalog-services/src/main/java/com/ca/mfaas/apicatalog/instance/InstanceInitializationException.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package com.ca.mfaas.apicatalog.instance;
+
+/**
+ * Exception used to indicate that the instance aren't initialized properly because of incorrect metadata.
```

Comment:
This exception is used to indicate the failed initialization of the instance caused by incorrect metadata.